smalldata_tools/lcls2/default_detectors.py

```diff
@@ -0,0 +1,330 @@
+# importing generic python modules
+import psana
+import abc
+import numpy as np
+try:
+    basestring
+except NameError:
+    basestring = str
+from smalldata_tools.common.detector_base import DefaultDetector_base
```

Comment:
This doesn't appear to be used anymore.

Similar to a previous comment, is there a reason the `default_detectors.py` cannot be combined into `common`? Without going through everything yet, I still see some `lcls2` prefaced detectors in `lcls1/default_detectors.py` (`lcls2_epics`..., `tt`, etc.). A factory could be used to return lcls1/lcls2 detectors which have the same name but different implementation?

Reply:
Work in progress to clean up / consolidate the default_detector setup. There are indeed some inconsistencies